k9mail-library/src/main/java/com/fsck/k9/mail/power/TracingPowerManager.java

```diff
@@ -1,16 +1,17 @@
 package com.fsck.k9.mail.power;
 
-import java.util.Timer;
-import java.util.TimerTask;
-import java.util.concurrent.atomic.AtomicInteger;
-
-import com.fsck.k9.mail.K9MailLib;
-
 import android.content.Context;
 import android.os.PowerManager;
 import android.os.PowerManager.WakeLock;
+import android.os.SystemClock;
 import android.util.Log;
 
+import com.fsck.k9.mail.K9MailLib;
```

Comment:
This change reorders the imports. Please import our [code style settings](https://github.com/k9mail/k-9/wiki/CodeStyle) so Android Studio will use our ordering.